README.md

```diff
@@ -0,0 +1,60 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Review Past Project
+
+> A project to microverse practice Code Review 2.0 training.
+
+## Built With
+
+- HTML
+- CSS
+- VS-Code
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Author
+
+👤 **Taofeek Abiodun Olalere**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ ] Since you've decided to add your social media handles, kindly supply the correct URL: the current ones are broken. e.g.
  - GitHub: [@teekaytech](https://github.com/teekaytech)